src/js/controllers/tx-details.js

```diff
@@ -1,20 +1,36 @@
 'use strict';
```

Comment:
There is way to much code in tx-details. js regarding this feature. it should be only a couple of lines.